homework/src/app.js

```diff
@@ -1,11 +1,254 @@
 {
   /* Your homework goes here */
-  
   /* STRINGS */
-
   /* ARRAYS */
-
   /* OPERATORS */
-
   /* QUESTIONS FROM HYF REPOS */
-}
\ No newline at end of file
+}
+
+// STRINGS
+// Strings Question 1
+
+//  Consider the following string:
+
+const myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+// Q: Add the string to your file and log it.
+
+// Solution
+
+console.log(myString);
+
+// Log the length of myString.
+
+// Solution : we use length property for the solution.
+
+console.log(myString.length);
+
+// Q:The commas make that the sentence is quite hard to read. Find a way to remove the commas from the string and replace them with spaces.
+
+// Solution
+
+const myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+let regex = /,/gi;
+
+console.log(myString.replace(regex, ' '));
```

Comment:
👍 but is there a reason why you used `let` instead of `const` while creating the `regex` variable?

Reply:
Dear Yash, thank  you for your valuable feedback. 
Actually , there is no specific reason why I used let instead of const. But I suppose it would be better if I use const instead of let because I do not need to re-assign the variable again.